module/VuFind/src/VuFind/Auth/AbstractBase.php

```diff
@@ -48,10 +48,12 @@
  * @link     https://vufind.org Main Page
  */
 abstract class AbstractBase implements
+    \VuFind\Db\Service\DbServiceAwareInterface,
```

Comment:
#3625 is going to remove DbTableAwareInterface/Trait from this class in favor of DbServiceAwareInterface/Trait. That is why I'm adding it in AbstractBase instead of the Shibboleth class, even though it's only presently used in one place.